Free Ruler/fi.lproj/MainMenu.strings

```diff
@@ -133,3 +133,6 @@
 
 /* Class = "NSMenuItem"; title = "Inches"; ObjectID = "lt1-Hj-2TR"; */
 "lt1-Hj-2TR.title" = "Tuumat";
+
+/* Class = "NSMenuItem"; title = "Cycle Units"; ObjectID = "2nm-aL-kZd"; */
+"2nm-aL-kZd.title" = "Vaihda Yksiköitä";
```

Comment:
@lanurmi This PR adds a menu item to cycle through the units pixels, millimeters, and inches. Can you review this localized string?